src/tests/sphmultipcqueue_t.c

```diff
@@ -840,8 +840,14 @@ fill_test_parallel_thread (void *arg)
 		CPU_ZERO_S(size,cset);
 		CPU_SET_S(cpu_list[tn],size,cset);
 		rc = sched_setaffinity(sphFastGetTID(),size,cset);
-		printf("%6d: sched_setaffinity(thread %d to CPU %d)=%d\n",
-		       sphFastGetTID(),tn,cpu_list[tn],rc);
+		if (rc == 0) {
+			printf("%6d: sched_setaffinity(thread %d to CPU %d)\n",
+				sphFastGetTID(),tn,cpu_list[tn]);
+		} else {
+			fprintf(stderr,"%6d: sched_setaffinity(thread %d to CPU %d) ",
+				sphFastGetTID(),tn,cpu_list[tn]);
+			perror(0);
+		}
```

Comment:
@ThinkOpenly   We have to do the same for sphdirectpcqueue_ttt
printf  needs some new line...
START 1 test thread 1 producers | 1 consumers (0x80000104000,30)
 21842: sched_setaffinity(thread 0 to CPU 8)  21843: sched_setaffinity(thread 1 to CPU 16) Invalid argument
Invalid argument